packages/block-editor/src/components/block-list-appender/index.js

```diff
@@ -94,11 +94,7 @@ function BlockListAppender( {
 			// Prevent the block from being selected when the appender is
 			// clicked.
 			onFocus={ stopPropagation }
-			className={ classnames(
-				'block-list-appender',
-				'wp-block',
```

Comment:
It was intentional, though thank you for checking. In trunk, there are two items having that class at the moment:

<img width="491" alt="Screenshot 2021-04-15 at 10 19 52" src="https://user-images.githubusercontent.com/1204802/114837595-2be53000-9dd4-11eb-96d1-259b2d7fa821.png">

For existing themes, those rules are pretty opinionated:

```
.editor-styles-wrapper .wp-block {
    margin-left: auto;
    margin-right: auto;
}
```

The fact that it works in trunk I'm not entirely sure why. But when tinkering with this, the rules started bleeding in, so best just to remove it instead of override.